docs/conventions/biome-configuration.md

```
@@ -20,14 +20,14 @@ The configuration follows an inheritance pattern where child configurations exte
 ## Schema Version
 
 The Biome configuration uses schema versions that should match the CLI version being used:
-- Root level: Schema version 2.1.1
+- Root level: Schema version 2.1.2
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)